internal/proxy/connection/manager.go

```diff
@@ -71,18 +42,19 @@ func (s *connectionManager) Stop() {
 func (s *connectionManager) checkLoop() {
 	defer s.wg.Done()
 
-	t := time.NewTicker(s.duration)
-	defer t.Stop()
+	t := time.NewTimer(paramtable.Get().ProxyCfg.ConnectionCheckInterval.GetAsDuration(time.Second))
```

Comment:
please use ticker.
refer to issue https://github.com/milvus-io/milvus/issues/22222

Reply:
do we need to refresh this ?

Reply:
If it is indeed necessary to refresh the interval, please consider using `ticker.Reset`.